HospitalSystem.ConsoleUI/Secretary/CrudUI.cs

```diff
@@ -0,0 +1,311 @@
+                }
+                else if (choice == "blocked patients" || choice == "bps")
+                {
+                    SelectBlockedPatients();
+                }
+                else if (choice == "block patient" || choice == "bp")
+                {
+                    BlockUserPatients();
+                }
+                else if (choice == "quit" || choice == "q")
+                {
+                    throw new QuitToMainMenuException("From StartManageEquipments");
+                }
+                else if (choice == "exit" || choice == "x")
+                {
+                    System.Environment.Exit(0);
+                }
+                else
+                {
+                    System.Console.WriteLine("INVALID INPUT - READ THE AVAILABLE COMMANDS!");
+                    System.Console.Write("INPUT ANYTHING TO CONTINUE >> ");
+                    ReadSanitizedLine();
+                }
+            }
+            catch (InvalidInputException e)
+            {
+                System.Console.Write(e.Message + " INPUT ANYTHING TO CONTINUE >> ");
+                ReadSanitizedLine();
+            }
+            catch (FormatException e)
+            {
+                System.Console.Write(e.Message + " INPUT ANYTHING TO CONTINUE >> ");
+                ReadSanitizedLine();
+            }
+        }
+    }
+
+    public void CreateUserPatient()
+    {   
+        Console.Clear();
+        UserService us = _hospital.UserService;
+
+        System.Console.WriteLine("Enter the following data: ");
+        System.Console.Write("email >> ");
+        string? email = Console.ReadLine();
+        if (email is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("password >> ");
+        string? password = Console.ReadLine();
+        if (password is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("first name >> ");
+        string? firstName = Console.ReadLine();
+        if (firstName is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("last name >> ");
+        string? lastName = Console.ReadLine();
+        if (lastName is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.WriteLine(email);
+        if(email == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+        }
+        else if(password == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+
+        }
+        else if(firstName == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+
+        }
+        else if(lastName == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+        }
```

Comment:
For real? If you wanted to do something like this add a wrapper class that takes input and does a back if input back. Like a ReadPagedCommand (dont call it that if you make it) that returns a string or throws a Returning to menu exception